### PR TITLE
Add attribution to accordion

### PR DESCRIPTION
**Pull request by davetaz**
Add attribution property to graphics on accordion. Brings this component in line with adapt-contrib-graphic.

### CODE REVIEW

**Comment by davetaz** [comment]
.graphic-attribution {
    font-size:0.75em;
    line-height: 1em;
} 

Is the style for this in adapt-contrib-graphic should the class be changed here and then this included in the less file?

**Comment by oliverfoster** [comment]
yes please, these components should be atomic, even it if means duplicates for the moment :+1:

**Comment by davetaz** [comment]
Closes adaptlearning/adapt_framework/#1526

**Comment by moloko** [comment]
would you mind quickly bumping the version to 2.0.6 in bower.json and README.md?

**Comment by moloko** [comment]
I know #82 has this already but I think that should probably be a separate release to this - and arguably needs to be 2.1.0

**Comment by davetaz** [comment]
bumped

File: less/accordion.less [hunk 1]
```diff
@@ -92,4 +92,8 @@
             vertical-align: bottom;
         }
     }
+    .accordion-item-graphic-attribution {
```
Comment (oliverfoster) [inline less/accordion.less]: would you mind doing ```.graphic-attribution``` as with the other one?

**Comment by sarveshwar-gavhane** [comment]
oh yes its missing

**Comment by davetaz** [comment]
added

**Pull request merged**